sensor/kubernetes/complianceoperator/handler_impl.go

```diff
@@ -790,13 +894,46 @@ func (m *handlerImpl) ctx() context.Context {
 	return concurrency.AsContext(&m.stopSignal)
 }
 
-func (m *handlerImpl) callWithRetry(fn func(context.Context) error) error {
+type retriableCall func(context.Context) error
+
+type onRetriableCallError func(context.Context, error) error
+
+type onConflictCall func(context.Context) error
+
+func onConflictErrorWrapper(fn onConflictCall) onRetriableCallError {
+	return func(ctx context.Context, parentErr error) error {
+		if !kubeAPIErr.IsConflict(parentErr) && !retry.IsRetryable(parentErr) {
+			return nil
+		}
+		return fn(ctx)
+	}
+}
```

Comment:
Let's maybe check if `fn` is not nil. Calling this func with a nil argument would panic.